engine/runtime-compiler/src/main/scala/org/enso/compiler/data/CompilerConfig.scala

```diff
@@ -4,14 +4,14 @@ import java.io.PrintStream
 
 /** Configuration for the compiler.
   *
-  * @param autoParallelismEnabled whether or not automatic parallelism detection
-  *                               is enabled.
-  * @param warningsEnabled whether or not warnings are enabled
-  * @param privateCheckEnabled whether or not private keyword is enabled
+  * @param autoParallelismEnabled     whether or not automatic parallelism detection
+  *                                   is enabled.
+  * @param warningsEnabled            whether or not warnings are enabled
+  * @param privateCheckEnabled        whether or not private keyword is enabled
   * @param staticTypeInferenceEnabled whether or not type inference is enabled
-  * @param irDumper Class name for the [[org.enso.compiler.dump.service.IRDumpService]] to use
-  *                 for IR dumping. If empty, no dumping will be performed.
-  * @param isStrictErrors if true, presence of any Error in IR will result in an exception
+  * @param irDumper                   Class name for the [[org.enso.compiler.dump.service.IRDumpFactoryService]] to use
+  *                                   for IR dumping. If empty, no dumping will be performed.
```

Comment:
Could the `enso.compiler.dumpIr` property specify FQN of a module to dump. Usually we are focus only one one module and we'd like to dump just that one. FQN would work for it well. Of course, various `*` or regex patterns may be supported as well, but FQN would be great start.